package.json

```diff
@@ -47,6 +48,7 @@
     "native-dns-packet": "^0.1.1",
     "nconf": "~0.6.9",
     "properties": "^1.2.1",
+    "redis": "0.12.x",
```

Comment:
`redis` is the node redis client.
`hiredis` is the binding to a C library of the same name, which is a very fast, minimal redis parser.

we need both if we want fast, nonblocking redis parsing, otherwise `redis` will just use a pure javascript parser.